Pod/Classes/XExtensionItem.h

```diff
@@ -96,10 +140,51 @@ typedef UIImage *(^XExtensionItemThumbnailProvidingBlock)(CGSize suggestedSize,
  */
 @property (nonatomic, copy) NSDictionary *userInfo;
 
+/**
+ Add entries from a dictionary-serializable custom object to this paramter object’s `userInfo` dictionary.
+ 
+ @param dictionarySerializable Object whose entries should be added to this paramter object’s `userInfo` dictionary.
+ */
+- (void)addEntriesToUserInfo:(id <XExtensionItemDictionarySerializing>)dictionarySerializable;
```

Comment:
I imagined that if the protocol name changed, the method name would have a corresponding change, like `- (void)addUserInfoParameterAddition:(id<XExtensionItemParameterAddition>)parameterAddition` (though this one obviously sucks). I think exposing userInfo still makes sense in case the user wants to add purely custom metadata, right?